File: Java/Car.java
class Car {
    Integer id;
    String license;
    Account driver;
    Integer passegenger;

    //declarando contructores
    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    //declaremos un metodo para que traiga lso datos del a clase
    void printDataCAr(){
        System.out.println("License: " + license + " Name Driver: " + driver.name);
    }
}

File: PHP/car.php
<?php
require_once "account.php";
class Car{
    public $id;
    public $license;
    public $driver;
    public $passenger;

    function __construct($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCAr(){
        echo "licence: $this->licence Driver: ".$this->driver->name;
    } 
}

?>